php8.0.Dockerfile

```diff
@@ -48,6 +48,19 @@ RUN \
 
 RUN echo extension=memcached.so >> /usr/local/etc/php/conf.d/memcached.ini
 
+# Install Mcrouter Extension
+RUN \
+  # Mcrouter Extension
+  git clone https://github.com/facebook/mcrouter.git && \
```

Comment:
Please make sure to pin a specfiic version so that upgrades won't break the library. You can see an example looking at the other extension installations